roles/openshift_master/tasks/main.yml

```diff
@@ -77,6 +81,25 @@
 - name: Install Master package
   yum: pkg={{ openshift.common.service_type }}-master{{ openshift_version  }} state=present
   register: install_result
+  when: not is_containerized | bool
+
+# TODO: enable when ansible#1993 lands and is widespread enough
+# - name: Docker image present
+#   docker:
+#     image: "{{ openshift.common.docker.image }}"
+#     state: image_present
+#   when: is_containerized | bool
```

Comment:
Of course It could. I originally wrote that as part of my openshift playbook that I run with a patched ansible.